week08/movie/src/Theme.js

```diff
@@ -0,0 +1,15 @@
+export const darkTheme = {
+  mode: "dark",
+  backgroundColor: "rgb(17, 17, 17)",
+  backgroundColor:
+    "linear-gradient(0deg, rgba(17, 17, 17, 1) 0%, rgba(34, 34, 34, 1) 45%, rgba(84, 84, 84, 1) 100%)",
+  Textcolor: "white",
+};
+
+export const lightTheme = {
+  mode: "light",
+  backgroundColor: "rgb(255,212,100)",
+  backgroundColor:
+    "linear-gradient(90deg, rgba(255,212,100,1) 0%, rgba(251,231,185,1) 33%, rgba(255,255,255,1) 100%)",
+  Textcolor: "black",
+};
```

Comment:
Theme.js에서 다크모드/라이트모드에 사용할 전역 색상을 정의해주시면 됩니다.
저는 예시로 기존 코드에서 Header 배경으로 사용한 색상과 slider 제목으로 사용한 색상을 정의해주었습니다.
이렇게 한 곳에 색상을 정의해두면 나중에 유지보수하기 편해진다는 장점이 있습니다.

```suggestion
export const darkTheme = {
  mode: "dark",
  // backgroundColor: "rgb(17, 17, 17)",
  backgroundColor:
    "linear-gradient(0deg, rgba(17, 17, 17, 1) 0%, rgba(34, 34, 34, 1) 45%, rgba(84, 84, 84, 1) 100%)",
  Textcolor: "white",
  header: "#222222",
  title: "#ffffff",
};

export const lightTheme = {
  mode: "light",
  // backgroundColor: "rgb(255,212,100)",
  backgroundColor:
    "linear-gradient(0deg, rgba(255,212,100,1) 0%, rgba(251,231,185,1) 33%, rgba(255,255,255,1) 100%)",
  Textcolor: "black",
  header: "#ffffff",
  title: "#f29b3e",
};

```